contract/src/main.rs

```diff
@@ -404,6 +416,50 @@ pub extern "C" fn set_variables() {
     }
 }
 
+fn add_page_entry_and_page_record(tokens_count: u64, item_key: &str) -> (u64, URef) {
```

Comment:
Lets move this function out into our utils module

Reply:
Moved thanks